packages/studio/src/lib/extensions/editor-camera/CameraControls.svelte

```diff
@@ -2,7 +2,7 @@
   module
   lang="ts"
 >
-  import CameraControls from 'camera-controls'
+  import { CameraControls } from 'camera-controls/dist/CameraControls.js';
```

Comment:
oh yeah thats right. FYI team, this change is here because we're waiting on an [external PR](https://github.com/yomotsu/camera-controls/issues/611) but can just use `pnpm patch` in the mean time.

Reply:
See above.